docker-hub/orgs.md

```diff
@@ -9,151 +9,228 @@ redirect_from:
 Docker Hub organizations let you create teams so you can give your team access
 to shared image repositories.
 
-- **Organizations** are collections of teams and repositories that can be managed together.
-- **Teams** are groups of Docker Hub users that belong to an organization.
+An **Organization** is a collection of teams and repositories
+that can be managed together. Docker users cannot belong directly to an
+organization. They belong only to teams within an organization. When you first
+create an organization, you’ll see that you have a team, the owners team, with a
+single member. An organization owner can then create new teams and add members
+to an existing team using Docker IDs or email address and by selecting a team
+the user should be part of.
+
+A **Team** is a group of Docker users that belong to an organization. An
+organization can have multiple teams. When you first create an organization,
+you’ll see that you have a team, the owners team, with a single member. An
+organization owner can then create new teams and add members to an existing team
+using Docker IDs and by selecting a team the user should be part of.
+
+> **Note:**
+>
+> In Docker Hub, users cannot belong directly to an organization.
```

Comment:
As mentioned above, I would say this wording could be ambiguous. "Directly" could be taken in different ways.

I'd word it that they require membership to at least one team.

Reply:
I believe this note is now redundant as the Org section (which is right above this) has the same info. I'll remove this note.